### ISSUE TITLE
md-tooltip not visible in md-autocomplete

### ISSUE THREAD

**Issue by robinsving** [angular/material#5356]
<h2>Issue</h2>
When using a md-autocomplete the inner html is replaced by an md-autocomplete-wrap. I can find no way of adding an md-tooltip with the latest version. See plunkr http://plnkr.co/edit/Ijq5YjBRzBcdpRQzhGMP?p=preview

<h2>Expected result:</h2>
Md-tooltip added as html is correctly shown when hovering the md-autocomplete

<h2>Simple snippet to start with:</h2>
```javascript
          return '\
        <md-autocomplete-wrap\
            layout="row"\
            ng-class="{ \'md-whiteframe-z1\': !floatingLabel, \'md-menu-showing\': !$mdAutocompleteCtrl.hidden }"\
            role="listbox">\
          ' + getAutocompleteTooltip() + '\
```
```javascript
      function getAutocompleteTooltip() {
        var templateTag = element.find('md-tooltip').detach(),
            html = templateTag.length ? templateTag.html() : element.html();
        if (!templateTag.length) element.empty();
        return '<md-tooltip>' + html + '</md-tooltip>';
      }
```

**Comment by SanderElias** [comment]
This is due to how md-auto-complete works. An easy workaround is something like this:

    <div>
        <md-autocomplete ng-disabled="ctrl.isDisabled" md-no-cache="ctrl.noCache" md-selected-item="ctrl.selectedItem" md-search-text="ctrl.searchText" md-items="item in ctrl.querySearch(ctrl.searchText)" md-item-text="item.display" placeholder="What is your favorite US state?">
            <span md-highlight-text="ctrl.searchText">{{item.display}}</span>
        </md-autocomplete>
        <md-tooltip>Tooltip should be here.</md-tooltip>
    </div>

Also, as md-auto-complete is an input element, it is from the standpoint of a11y an better idea to put in an aria-label and/or a placeholder text.

**Comment by AnkitGupta88** [comment]
In this case, tooltip is not coming everytime. it will disappear when we move onto another item of mdvirtual repeat. [Pop up]
